source/Irrlicht/CEGLManager.cpp

```diff
@@ -590,6 +594,19 @@ const SExposedVideoData& CEGLManager::getContext() const
 	return Data;
 }
 
+void* CEGLManager::getProcAddress(const std::string name)
+{
+	void* proc = NULL;
+	proc = (void*)eglGetProcAddress(name.c_str());
+	if (!proc) { // fallback

```

Comment:
Do we actually know if this is needed or is it here just in case?

Reply:
SDL does it so I do it as well, it doesn't hurt in any case.